wiki-webui/src/main/java/org/exoplatform/wiki/webui/control/action/ExportAsPDFActionComponent.java

```diff
@@ -120,5 +128,29 @@ private File createPDFFile(String title, String content) throws IOException {
       }
       return pdfFile;
     }
+    private String htmlToxhtmlAndBase64Images(Page page) throws WikiException {
+      WikiService wikiService = CommonsUtils.getService(WikiService.class);
+      org.jsoup.nodes.Document doc = Jsoup.parse(page.getContent());
+      doc.outputSettings().syntax(org.jsoup.nodes.Document.OutputSettings.Syntax.xml);
+      Elements elements = doc.getAllElements();
+      for(Element element : elements){
+        if(element.tagName().equals("img") && !element.attr("src").isEmpty()){
+          String src = element.attr("src");
+          String imgName = src.substring(src.lastIndexOf("/") + 1);
+          Attachment imageAttachment = wikiService.getAttachmentOfPageByName(imgName, page, true);
+          if(imageAttachment == null) {
+            Page parentPage = wikiService.getParentPageOf(page);
+            imageAttachment = wikiService.getAttachmentOfPageByName(imgName, parentPage, true);
+          }
+          if(imageAttachment != null && imageAttachment.getMimeType().startsWith("image/")){
```

Comment:
```suggestion
          if(imageAttachment != null && imageAttachment.getMimeType() != null && imageAttachment.getMimeType().startsWith("image/")){
```
something missing here should be in PR 261